apps/emqx_bridge_kafka/src/emqx_bridge_kafka.erl

```diff
@@ -380,7 +380,7 @@ fields(consumer_kafka_opts) ->
             )},
         {offset_commit_interval_seconds,
             mk(
-                pos_integer(),
+                emqx_schema:timeout_duration_s(),
                 #{default => 5, desc => ?DESC(consumer_offset_commit_interval_seconds)}
```

Comment:
Now that the type has changed, we need to change the default value:

```suggestion
                #{default => <<"5s">>, desc => ?DESC(consumer_offset_commit_interval_seconds)}
```

Reply:
Ack